### ISSUE TITLE
[in the name of] propose to organization to make requests in the name of their users

### ISSUE THREAD

**Issue by jibe-b** [OAButton/discussion#916]
Up to now, requests are made by individuals.

Batch upload and existence of increasing number of digital OA tools make several organization potential proxies between their users (which are not OAB users) and the authors of articles.

For example, an organization that provide an OpenAccess boosting DOI resolver sees its users ask for DOIs. People in this organization could then propose to their users to make a request for each publication that is not yet available in an open archive. See https://github.com/Impactstory/oadoi/issues/40 for the concrete case inspiring this example.

It could also be an OpenAccess journal that wants its readers to have access to the knowledge cited in the bibliography.

How it would be organized is:
- organization register as "organizations" to OAB
- these organizations set up a few things like stories to send to the authors
- then send regular tables with DOIs for which they want to make requests
- a twitter bot (and RSS feed) tweets with a hashtag and the name of the organization whenever a publication is made available
- the organizations provide a feed (twitter, RSS, directly on webpage) for users to know when publications are made available (not only the one they searched for but all)

This process requires little programming, mostly communication: "organizations are invited to make requests in their own names and be a proxy for their users: they ask for DOIs with which users have had an interaction and they provide a feed for their users to be notified."

**Comment by nemobis** [comment]
I agree this is something worth setting up/exploring. Maybe, to avoid overcomplicating it, you might want to start with something that one university is willing to actively use. If you are able to set up something, even very simple, I expect there will be interested from at least one of the universities I'm in contact with re https://lists.wikimedia.org/pipermail/openaccess/2017-September/000252.html

**Comment by jibe-b** [comment]
Thanks @nemobis

I was at first thinking about organizations like ImpactStory with OADOI which have logs of DOIs and can speak in the name of their users.

You make me think about the OpenURL resolvers (including SFX), which are linked to a research institution. The librarians that manage this system could take care of collecting DOIs and making requests in the name of the researchers.



And it's worth doing it also keeping the wo.man in the loop. So whenever I find an option to have this "send in the name of" possible directly with people, I'll tell you ;)

**Comment by jibe-b** [comment]
Related https://twitter.com/ryregier/status/926117509216047104
